README.md

```diff
@@ -28,57 +33,102 @@ Project DeepSpeech is an open source Speech-To-Text engine. It uses a model trai
 
-After the library files are installed, you can use pip to install the Python package, like so:
+```bash
+LD_LIBRARY_PATH=. ./deepspeech -h
+```
+
+You can consult the [native client README](native_client/README.md) for more details.
+
+### Installing Python bindings
+
+Pre-built binaries that can be used for performing inference with a trained model can be found on TaskCluster. You'll need to download the appropriate Python wheel package.
+
+[deepspeech-0.0.1-cp27-cp27mu-linux_x86_64.whl (Python 2.7, Linux / amd64)](https://index.taskcluster.net/v1/task/project.deepspeech.deepspeech.native_client.master.cpu/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_x86_64.whl)
+[Other configurations](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/project.deepspeech.deepspeech.native_client.master)
+
+You can use pip to install the Python package, like so:
 ```bash
 pip install <path to .whl file>
 ```
 
-### Installing DeepSpeech Python bindings from source
+See [client.py](native_client/client.py) for an example of how to use the bindings.
+
+### Installing Node.JS bindings
+
+You can download the Node.JS bindings using `util/tc.py` and install them with `npm`:
+
+```bash
+python util/tc.py --target . --artifact deepspeech-0.0.1.tgz
+npm install deepspeech-0.0.1.tgz
+```
+
+See [client.js](native_client/client.js) for an example of how to use the bindings.
+
+### Installing bindings from source
 
 If pre-built binaries aren't available for your system, you'll need to install them from scratch. Follow [these instructions](native_client/README.md).
 
-### Installing other requirements
+## Training
+
+### Installing prerequisites for training
+
+Install the required dendencies using pip:
 
-Manually install [Git Large File Storage](https://git-lfs.github.com/), then open a terminal and run:
 ```bash
-git clone https://github.com/mozilla/DeepSpeech
 cd DeepSpeech
 pip install -r requirements.txt
 ```
 
-## Recommendations
+You'll also need to download `native_client.tar.xz` or build the native client files yourself to get the custom TensorFlow OP needed for decoding the outputs of the neural network. You can use `util/tc.py` to download the files for your architecture:
 
-If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommended to install TensorFlow with GPU support. Training will likely be significantly quicker than using the CPU.
+```bash
+python util/tc.py destination/folder cpu
+```
 
-## Training a model
+This will download the native client files for the x86_64 architecture without CUDA support, and extract them into `destination/folder`. If you prefer building the binaries from source, see the [native_client README file](native_client/README.md). We also have binaries with CUDA enabled ("gpu") and for ARM7 ("arm").
 
-The central (Python) script is `DeepSpeech.py` in the project's root directory. For its list of command line options, you can call:
+### Recommendations
```

Comment:
This doesn't use our TensorFlow. This will not guarantee ABI/API compatibility with our op. So I don't think this is the preferred means to install TensorFlow.